anomalib/utils/cli/cli.py

```diff
@@ -213,23 +215,37 @@ def __set_callbacks(self) -> None:
         add_visualizer_callback(callbacks, config)
         self.config[subcommand].visualization = config.visualization
 
-        # TODO: https://github.com/openvinotoolkit/anomalib/issues/19
-        if config.openvino and config.nncf:
-            raise ValueError("OpenVINO and NNCF cannot be set simultaneously.")
-
         # Export to OpenVINO
-        if config.openvino:
-            from anomalib.utils.callbacks.openvino import (  # pylint: disable=import-outside-toplevel
-                OpenVINOCallback,
+        if "openvino" in config.export_mode:
+            from anomalib.utils.callbacks.export import (  # pylint: disable=import-outside-toplevel
+                ExportCallback,
+            )
+
+            logger.info("Setting model export to OpenVINO")
+            callbacks.append(
+                ExportCallback(
+                    input_size=config.data.init_args.image_size,
+                    dirpath=os.path.join(config.trainer.default_root_dir, "compressed"),
+                    filename="model",
+                    export_mode="openvino",
+                )
+            )
+        elif "onnx" in config.export_mode:
+            from anomalib.utils.callbacks.export import (  # pylint: disable=import-outside-toplevel
+                ExportCallback,
             )
 
+            logger.info("Setting model export to ONNX")
             callbacks.append(
-                OpenVINOCallback(
+                ExportCallback(
                     input_size=config.data.init_args.image_size,
                     dirpath=os.path.join(config.trainer.default_root_dir, "compressed"),
                     filename="model",
+                    export_mode="onnx",
                 )
             )
+        else:
+            logger.info("No model export")
```

Comment:
Same duplication here

Reply:
addressed in: [237f9ff](https://github.com/openvinotoolkit/anomalib/pull/509/commits/237f9ffedd31499f1b1779e2206cbed600161171)